base/special/gamma.jl

```diff
@@ -27,9 +27,10 @@ lgamma_r(x::Number) = lgamma(x), 1 # lgamma does not take abs for non-real x
 """
     lfact(x)
 
-Compute the logarithmic factorial of `x`
+Compute the logarithmic factorial of a nonnegative integer `x`.
+Equivalent to [`lgamma`](@ref) of `x + 1`.
```

Comment:
Once CI finishes I'll push the commit that contains that change with a `[ci skip]`. When this is merged, it should be squashed and the skip should be removed.